webapp/channels/src/components/settings_sidebar/settings_sidebar.tsx

```diff
@@ -110,17 +110,31 @@ export default class SettingsSidebar extends React.PureComponent<Props> {
             pluginTabList = (
                 <>
                     <hr/>
-                    <li
-                        key={'plugin preferences heading'}
-                        role='heading'
-                        className={'header'}
+                    <div
+                        aria-labelledby={'userSettingsModal.pluginPreferences.header'}
+                        role='region'
                     >
-                        <FormattedMessage
+                        <div
                             id={'userSettingsModal.pluginPreferences.header'}
-                            defaultMessage={'PLUGIN PREFERENCES'}
-                        />
-                    </li>
-                    {this.props.pluginTabs.map((tab, index) => this.renderTab(tab, index))}
+                            key={'plugin preferences heading'}
+                            role='heading'
+                            aria-level={3}
+                            className={'header'}
+                        >
+                            <FormattedMessage
+                                id={'userSettingsModal.pluginPreferences.header'}
+                                defaultMessage={'PLUGIN PREFERENCES'}
```

Comment:
I know this was this way before, but we should try to always keep the texts lower case and capitalize only when necessary via css.

Reply:
I didn't want to go beyond the issue's scope, so I haven't updated it. However, if you'd like, I can fix this.

Reply:
We can leave it as-is. If we change it, it'll make translators have to retranslate it as the lowercase version